torch/distributed/distributed_c10d.py

```diff
@@ -559,6 +570,11 @@ def _new_process_group_helper(world_size,
         raise RuntimeError("Expected timeout argument to be of type"
                            "datetime.timedelta")
 
+    if pg_options is not None and timeout != default_pg_timeout and pg_options.timeout != timeout:
```

Comment:
so if pg_options has a non-default timeout set, but user did not set timeout and thus timeout == default_pg_timeout, it looks like we override it with the pg_options timeout? This is probably fine, but would be good to test and document this.

Reply:
yes, sounds good let me add this behavior to the doc.